### ISSUE TITLE
[Spell] Healing Stream Totem

### ISSUE THREAD

**Issue by Dranah** [WoWManiaUK/Redemption#2831]
https://www.wowhead.com/spell=5394/healing-stream-totem
This broke recently.

**How it works atm:** The totem is casting the Healing Stream effect http://www.wow-mania.com/armory/?spell=58765 on the shaman instead of itself.
**How it should work:** The Healing Stream effect is meant to be a self cast (0 yrd) ability that the totem casts on itself. This effect then heals all nearby (30yrd) party members every 2sec when it ticks.

This bug has 2 consequences:
1) Annoying visual issue that alerted me of the bug in the first place, the shaman is casting the animation every 2seconds when the totem should tick, instead of the totem flashing up and pulsing as it used to.
2) Since the effect is cast on the shaman instead of the totem, the utility of the totem suffers from range issues: 
- If the shaman is not within 30 yards of the totem, there is no healing effect at all, because the totem can't reach the shaman to cast healing stream on him. (normally the totem would heal nearby party members no matter where the shaman is)
- If the shaman is within 30 yard of the totem, but the party members are not in 30 yard of the shaman (i.eg. theyre on two different sides of the totem), the totem will not heal party members because the Healing Stream effect calculates range from the shaman, not the totem, due to being on wrong target. 
Long story short, if the effect is not on the totem, then being in range of the totem does not mean you will receive any healing, which is wrong.

**Comment by Thrashattack** [comment]
I experienced this while leveling my shaman.

**Comment by Keader** [comment]
Test tomorrow, upload server looks off xd

**Comment by Thrashattack** [comment]
Update: The totem disappear after 2 minutes, should be 5. 
The effect of heals persists because it doesnt comes from the totem. 
![image](https://user-images.githubusercontent.com/23486776/47610157-2c2ebc80-da1c-11e8-96b4-4b1bbc0d61f1.png)

**Comment by Dranah** [comment]
I think the fix has been built only now.
UPDATE:
The totem now works as intended, the healing stream effect is self cast on the totem, visual pulse is displayed by the totem and the healing range is calculated properly from the totem.
It also lasts 5minutes as intended.
